app/views/training/pages/_content_wrap.html.slim

```diff
@@ -1,6 +1,6 @@
 .govuk-grid-row
```

Comment:
Take a look at https://github.com/DFE-Digital/early-years-foundation-recovery/pull/826/files where the nested page content is deleted.